jmh-benchmarks/src/main/java/org/apache/kafka/jmh/fetcher/ReplicaFetcherThreadBenchmark.java

```diff
@@ -166,7 +166,7 @@ public void setup() throws IOException {
             TransferLeaderManager transferLeaderManager = Mockito.mock(TransferLeaderManager.class);
             Partition partition = new Partition(tp, 100, ApiVersion$.MODULE$.latestVersion(),
                     0, Time.SYSTEM, isrChangeListener, new DelayedOperationsMock(tp),
```

Comment:
Oh, there's the Clock object in the Partition ctor => you should use it in the main code instead of System.currentTimeMillis(). It has a method with the same semantics.

Reply:
Good catch! Will address.